guide/invoicing/overview.mdx

```diff
@@ -59,3 +59,93 @@ You can specify multiple `grouped_by` keys for your charge; the order you set th
 The `grouped_by` feature is exclusively available for the `standard` charge model.
 
 </Note>
+
+
+## Finalize empty invoices
+By default, Lago finalizes all invoices, including those with a final amount of zero. However, some customers may prefer to skip empty invoices to avoid unnecessary accounting documents. 
+In such cases, you **can configure Lago to skip the issuance of invoices when they are empty**.
+
+### Empty invoices definition
+Lago considers an invoice empty only if it has no line items before applying coupons or discounts. 
+If an applicable discount reduces an invoice's total to zero, Lago will still finalize that invoice.
```

Comment:
If an applicable discount reduces an invoice’s total to zero, Lago will not consider this invoice as part of the empty invoices scope and will still finalize it.